src/components/CodeEditor/styles.css

```diff
@@ -0,0 +1,4 @@
+:global .CodeMirror {
+    font-size: 13px;
```

Comment:
You can put this in the code editor component, I'll comment there.

Reply:
Try it again with something like this:

https://github.com/taskcluster/taskcluster-web/blob/master/src/App/index.jsx#L23

```js
'@global': {
  '.CodeMirror': {
    fontSize: 13,
    height: '100%',
  },
},
```